openedx/core/djangoapps/appsembler/api/v1/views.py

```diff
@@ -77,6 +79,7 @@ class TahoeAuthMixin(object):
 
 class RegistrationViewSet(TahoeAuthMixin, viewsets.ViewSet):
 
```

Comment:
Can we get something like this in the API class docstring?

```suggestion
"""
Registration tada yada...

This API endpoint is throttled at the rate of 60 requests per minute. Reaching the limit will cause the API to issue HTTP code `429` (Too many requests). Please allow your API client to handle that case, and be mindful of keep the rate below 60 per minute.

Ta da ya da.
"""
```